### DISCUSSION TITLE
Lack of support language from predefined recognizers.

### DISCUSSION THREAD

**Discussion by xsandiii** [microsoft/presidio#1389] [Requested features]
We encountered the issue with detecting CreditCard using CreditCardRecognizer.
In the config file default_recognizers.yaml you clearly defined the supported languages:
  - name: CreditCardRecognizer
    supported_languages:
    - language: en
      context: [credit, card, visa, mastercard, cc,
      amex, discover, jcb, diners, maestro, instapayment]
    - language: es
      context: [tarjeta, credito, visa, mastercard, cc,
      amex, discover, jcb, diners, maestro, instapayment]
    - language: it
    - language: pl
    type: predefined

But during tests, the credit card with language ES is not possible to detect with recognizer, it looks like,
 the features does not work properly.

The API request with given 'es' language:
{
    "text": "tarjeta 4012 8888 8888 1881",
    "language": "es"
}

Response:
[
    {
        "analysis_explanation": null,
        "end": 7,
        "entity_type": "LOCATION",
        "recognition_metadata": {
            "recognizer_identifier": "SpacyRecognizer_139939461637360",
            "recognizer_name": "SpacyRecognizer"
        },
        "score": 0.85,
        "start": 0
    }
]

The API request with given 'en' language:
{
    "text": "tarjeta 4012 8888 8888 1881",
    "language": "en"
}

Response:
[
    {
        "analysis_explanation": null,
        "end": 27,
        "entity_type": "CREDIT_CARD",
        "recognition_metadata": {
            "recognizer_identifier": "CreditCardRecognizer_139939461637216",
            "recognizer_name": "CreditCardRecognizer"
        },
        "score": 1.0,
        "start": 8
    },
    {
        "analysis_explanation": null,
        "end": 27,
        "entity_type": "DATE_TIME",
        "recognition_metadata": {
            "recognizer_identifier": "SpacyRecognizer_139939356647376",
            "recognizer_name": "SpacyRecognizer"
        },
        "score": 0.85,
        "start": 23
    }
]

Please could you explain me, if this is bug?

**Comment by omri374** [comment]
Hi @xsandiii, the support for a yaml based config is not yet released. Are you testing this with the code in github or through the released Docker/pypi packages?

cc @RoeyBC @SharonHart

**Reply by xsandiii** [reply]
Thank you for the response. Yes, we are working in GitHub, on forked version. Finally, we did find the solution to the issue by ourselves. We needed to define another supported languages in presidio-analyzer/presidio_analyzer/conf/default_recognizers.yaml.
However, many thanks for your fast reaction.
